brambling/templatetags/brambling.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+from django.utils.encoding import force_text
+
+register = template.Library()
+
+
+@register.filter
+def to_string(val):
```

Comment:
Oh, I forgot this wasn't a builtin. Django does have the |stringformat filter, which could be used to the same effect (i.e. converting an int to a string) – might be better to do that rather than writing a new filter just for this purpose.